StereoKit/Native/NativeTypes.cs

```diff
@@ -242,7 +242,13 @@ public struct UISettings
 		/// <summary>How wide is the back-border around the UI elements? In
 		/// meters.</summary>
 		public float backplateBorder;
-	}
+		/// <summary>
+		/// Defines the scale factor for the horizontal separator's thickness.
+		/// The thickness is calculated by multiplying the height of the text
+		/// by this factor. The default valus is 0.4f.
+		/// </summary>
+		public float hseparatorScale;
```

Comment:
Perhaps it would be better to make it not specific to horizontal separators? When there is also vertical separator component, it would make sense to have the same factor for both. So I would rename to `separatorScale`/`separator_scale`, and remove the "horizontal" wording in the doc...